cmd/warpforge/main_test.go

```diff
@@ -49,6 +50,21 @@ func testFile(t *testing.T, fileName string, workDir *string) {
 	doc.BuildDirIndex()
 	patches := testmark.PatchAccumulator{}
 	for _, dir := range doc.DirEnt.ChildrenList {
+		if _, hasNetTests := dir.Children["net"]; hasNetTests {
+			if *testutil.FlagOffline {
+				t.Log("skipping test", dir.Name, "due to offline flag")
+				continue
+			}
+			// we want to run the contents of the `/net` dir
+			t.Run(dir.Name, func(t *testing.T) {
+				test := testexec.Tester{
+					ExecFn:   execFn,
+					Patches:  &patches,
+					AssertFn: assertFn,
+				}
+				test.Test(t, *dir.Children["net"])
+			})
```

Comment:
Surely you don't need to duplicate all this just to change what the function call is applied on?

Reply:
Surely you are correct.